src/NBitcoin/Network.cs

```diff
@@ -327,6 +283,12 @@ public PubKey AlertPubKey
 
         public string Name { get; private set; }
 
+        /// <summary> The name of the root folder containing the different blockchains. </summary>
```

Comment:
Maybe we can elaborate on the description?
"The root folder represents blockchains of the same consensus rules, for example testnet, regtest, segwit(testchain) etc.."